ecs-cli/modules/commands/compose/service/compose_service_command.go

```diff
@@ -189,3 +193,14 @@ func loadBalancerFlags() []cli.Flag {
 		},
 	}
 }
+
+// ComposeServiceTimeOutFlag allows user to specify a custom timeout
+func ComposeServiceTimeOutFlag() cli.Flag {
```

Comment:
Argh, I hate to do this to you, but since the flag is `timeout` as oneword, can this be `ComposeServiceTimeoutFlag`?